__tests__/app.test.js

```diff
@@ -82,6 +84,34 @@ describe('GET /api/reviews/:review_id', () => {
             expect(msg).toBe('No review found for review_id: 100');
         });
     });
+    
+    test('200: specified review response should include comment_count property with value of 0', () => {
```

Comment:
These tests are both covering the same thing, and that's not DRY - therefore, I think you should make one, more generic, test along the lines of "should include comment_count property with the correct value for the review_id passed"